src/models/item.js

```diff
@@ -0,0 +1,65 @@
+const mongoose = require('mongoose');
+const isImageUrl = require('is-image-url');
```

Comment:
After a lot of search for regex expressions and other ways to check for image URL, I found this package which has a high accuracy rate. I tested its return value manually; it does an excellent job compared to other solutions but unfortunately there is one URL I entered that is not an image and it returned true. But overall, it is the best solution that I have found

Reply:
@AmmarBaki2 Good work on researching and finding an effective solution. I checked and this package does look reliable. One small issue could be that it was last updated 6 years ago, but it still has 4000 weekly downloads so looks like people are using it.

Reply:
So, what should I do? Should I leave it?

Reply:
Yup, leave it as it is for now.

Even though there are a few other similar libraries that were updated more recently, they don't have as many downloads as of this one. At least you've tested this out and found high accuracy. For now, let's proceed with other issues on the project. If later we have more time we can look for better libraries.

(Or consider this a future project and build your own library. 😄 )